catroid/src/main/res/values/strings.xml

```diff
@@ -1737,36 +1735,38 @@
-    <string name="formula_editor_function_arduino_read_pin_value_analog">arduino_analog_pin</string>
-    <string name="formula_editor_function_raspi_read_pin_value_digital">raspberry_pi_pin</string>
+    <string name="formula_editor_function_arduino_read_pin_value_digital">arduino digital pin</string>
+    <string name="formula_editor_function_arduino_read_pin_value_analog">arduino analog pin</string>
+    <string name="formula_editor_function_raspi_read_pin_value_digital">raspberry pi pin</string>
     <string name="formula_editor_function_pin_default_parameter">(3)</string>
-    <string name="formula_editor_function_finger_x">screen_touch_x</string>
-    <string name="formula_editor_function_finger_y">screen_touch_y</string>
-    <string name="formula_editor_function_is_finger_touching">screen_is_touched</string>
-    <string name="formula_editor_function_multi_finger_x">screen_touch_x</string>
-    <string name="formula_editor_function_multi_finger_y">screen_touch_y</string>
-    <string name="formula_editor_function_is_multi_finger_touching">screen_is_touched</string>
-    <string name="formula_editor_function_index_of_last_finger">last_screen_touch_index</string>
+    <string name="formula_editor_function_finger_x">screen touch x</string>
+    <string name="formula_editor_function_finger_y">screen touch y</string>
+    <string name="formula_editor_function_is_finger_touching">screen is touched</string>
+    <string name="formula_editor_function_multi_finger_x">screen touch x</string>
+    <string name="formula_editor_function_multi_finger_y">screen touch y</string>
+    <string name="formula_editor_function_is_multi_finger_touching">screen is touched</string>
+    <string name="formula_editor_function_index_of_last_finger">last screen touch index</string>
     <string name="formula_editor_function_no_parameter"></string>
     <string name="formula_editor_function_touch_parameter">(1)</string>
     <string name="formula_editor_function_list_item">element</string>
     <string name="formula_editor_function_list_item_parameter">(1,*list name*)</string>
     <string name="formula_editor_function_contains">contains</string>
     <string name="formula_editor_function_contains_parameter">(*list name*,1)</string>
-    <string name="formula_editor_function_collision">touches_object</string>
-    <string name="formula_editor_function_collides_with_edge">touches_edge</string>
-    <string name="formula_editor_function_touched">touches_finger</string>
-    <string name="formula_editor_sensor_x_acceleration">acceleration_x</string>
-    <string name="formula_editor_sensor_y_acceleration">acceleration_y</string>
-    <string name="formula_editor_sensor_z_acceleration">acceleration_z</string>
-    <string name="formula_editor_sensor_compass_direction">compass_direction</string>
+    <string name="formula_editor_function_number_of_items">number of elements</string>
```

Comment:
```suggestion
    <string name="formula_editor_function_number_of_items">number of items</string>
```
Sorry for the back and forth, but I checked once more, and in Scratch it is called "items", and also we decided in 2014 to rename the "element" function to "item", see https://jira.catrob.at/browse/CAT-2374